provider/pkg/gen/examples/overlays/configGroupV2.md

```diff
@@ -0,0 +1,513 @@
+ConfigGroup creates a set of Kubernetes resources from Kubernetes YAML text. The YAML text
```

Comment:
For the file path, the v2 versions are not overlays, right?

Reply:
That's true but neither is `Release` and yet we see `helmRelease.md` in `overlays/`. Similarly, we see release stuff in `provider/pkg/gen/overlays.go`.

The term "overlay" seems to have a secondary meaning in the kubernetes provider, meaning "not generated from the openapi spec".  I would prefer to leave it as-is, is that OK?